test/contracts/apps/disputable/disputable_app.js

```diff
@@ -67,17 +69,19 @@ contract('DisputableApp', ([_, owner, agreement, anotherAgreement, someone]) =>
 
       const itSetsTheAgreementAddress = agreement => {
         it('sets the agreement', async () => {
-          await disputable.setAgreement(agreement, { from })
+          await dao.setAgreement(disputable.address, agreement, { from })
 
           const currentAgreement = await disputable.getAgreement()
           assert.equal(currentAgreement, agreement, 'disputable agreement does not match')
         })
 
         it('emits an event', async () => {
-          const receipt = await disputable.setAgreement(agreement, { from })
+          const { tx } = await dao.setAgreement(disputable.address, agreement, { from })
+          const receipt = await web3.eth.getTransactionReceipt(tx)
```

Comment:
Oh interesting, was the receipt we got as the return of `dao.setAgreement()` not sufficient?

Reply:
For some reason, it wasn't working. I think this will be fixed when we upgrade to Truffle 5 + helpers